extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1348,6 +1351,27 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Checks if the page has any captions and toggles them.
+   * @private
+   */
+  checkPageHasCaptions_() {
```

Comment:
nit: Should we add toggle to the function name? I know it's in the description above but it would be helpful when seeing the function name elsewhere. Maybe something like `checkTogglePageCaptions`?